docs/using-hive.md

```diff
@@ -558,6 +560,73 @@ openstack:
   flavor: m1.large
 ```
 
+#### Configuring Availability Zones
+
+The desired Availability Zones (AZ) to create new worker nodes in can be specified in the `MachinePool` YAML (`spec.platform.<provider>.zones`), for example:
```

Comment:
> specified in the `MachinePool` YAML (`spec.platform.<provider>.zones`),

might want to reword it on the lines of ``specified in the `MachinePool` YAML usually `spec.platform.<provider>.zones`,`` to make it clear that different platforms can have different paths for this value.